ui/components/app/modals/convert-token-to-nft-modal/convert-token-to-nft-modal.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+import Modal from '../../modal';
+import Typography from '../../../ui/typography';
+import {
+  TYPOGRAPHY,
+  FONT_WEIGHT,
+  ALIGN_ITEMS,
+  DISPLAY,
+} from '../../../../helpers/constants/design-system';
+import Box from '../../../ui/box';
+import withModalProps from '../../../../helpers/higher-order-components/with-modal-props';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { ADD_COLLECTIBLE_ROUTE } from '../../../../helpers/constants/routes';
+
+const ConvertTokenToNFT = ({ hideModal, tokenAddress }) => {
+  const history = useHistory();
+  const t = useI18nContext();
+  return (
+    <Modal
+      onSubmit={() => {
+        history.push({
+          pathname: ADD_COLLECTIBLE_ROUTE,
+          state: { tokenAddress },
+        });
+        hideModal();
+      }}
+      submitText={t('yes')}
+      onCancel={() => hideModal()}
+      cancelText={t('cancel')}
+      contentClass="convert-token-to-nft-content"
+      containerClass="convert-token-to-nft-container"
+    >
+      <div className="convert-token-to-nft">
+        <Box
+          marginTop={2}
+          display={DISPLAY.INLINE_FLEX}
+          alignItems={ALIGN_ITEMS.CENTER}
+        >
```

Comment:
Weird question but `<Typography>` component takes a `boxProps` prop; could we use that prop instead of wrapping with a `<Box>`?  Just curious.

Reply:
yes good call. Updated here https://github.com/MetaMask/metamask-extension/pull/13291/commits/cf380c05a5d45a435f90101ab9d03e4dbf6a347f